db/mysql-instance.yml

```diff
@@ -146,42 +146,31 @@ Resources:
         - Fn::ImportValue:
             !Sub ${VPCStack}-Subnet2
   MasterDB:
-    Type: 'AWS::RDS::DBInstance'
+    Type: "AWS::RDS::DBInstance"
+    DependsOn: RDSSubnetGroup
     Properties:
 # CloudFormation cannot update a stack when a custom-named resource requires replacing,
```

Comment:
I guess we also should remove this comment, since we don't really want people to use a preexisting stack to do snapshot recoveries